src/hooks/index.tsx

```diff
@@ -74,7 +74,7 @@ export async function getFileContent(
   const { repo, owner, path, fileRef } = params;
   const branch = fileRef || "HEAD";
 
-  const apiUrl = `https://raw.githubusercontent.com/${owner}/${repo}/${branch}/${path}`;
+  const apiUrl = `https://api.github.com/repos/${owner}/${repo}/contents/${path}?ref=${branch}`;
   const res = await fetch(apiUrl, {
     headers: PAT ? { Authorization: `token ${PAT}` } : {},
   });
```

Comment:
I think you need to unpack the `content` field from the response and base64-decode it, as we do in `getFileContent` in `blocks`. Or maybe there is a way to get raw content by giving a content-type header, see https://docs.github.com/en/rest/reference/repos#custom-media-types-for-repository-contents

Reply:
this is actually already returning raw content (I swear I tested it this time!)

Reply:
I'll add the header though, just to make sure!

Reply:
ha ha I believe you! but I am confused, is this not the same endpoint we call in `getFileContent`? there we call it through Axios, maybe Axios sets a header that changes the response?

Reply:
through curl I could only get a raw response with an `Accept` header, e.g.
```
curl -H "Accept: application/vnd.github.v3.raw" https://api.github.com/repos/githubnext/blocks-tutorial/contents/README.md
```

Reply:
yeah, super strange! I'm not sure how that was working for me. Lmk if the changes work for you!